src/Compilers/Core/MSBuildTask/InteractiveCompiler.cs

```diff
@@ -194,17 +216,15 @@ protected override int ExecuteTool(string pathToTool, string responseFileCommand
 
         protected override string GenerateCommandLineCommands()
         {
-            var commandLineBuilder = new CommandLineBuilderExtension();
-            AddCommandLineCommands(commandLineBuilder);
-            return commandLineBuilder.ToString();
+            return DotnetHostInfo.CommandLineArgs;
```

Comment:
Sort of, yes. I guess this is the whole point of the majority of this change - to transform `(cmd: "csc.dll", args: "...")` to `(cmd: "dotnet", args: "csc.dll ...")`, while conforming to the `ToolTask` API.

`Csc.CommandLineArgs` itself will not contain it (as that is the plain arguments passed into the task), but `GenerateCommandLineCommands()`->`ExecuteTool()` will.